File: esthesis-server/esthesis-server-impl/src/main/java/esthesis/platform/backend/server/dto/CampaignMemberDTO.java
package esthesis.platform.backend.server.dto;

import esthesis.platform.backend.common.dto.BaseDTO;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;
import lombok.experimental.Accessors;

@Data
@NoArgsConstructor
@Accessors(chain = true)
@EqualsAndHashCode(callSuper = true)
public class CampaignMemberDTO extends BaseDTO {
  private String identifier;
  private int groupOrder;
  private int type;
}

File: esthesis-ui/src/app/dto/campaign-member-dto.ts
export class CampaignMemberDto {
  type: number;
  identifier: string;
  groupOrder: number;

  constructor(type: number, identifier: string, groupOrder: number) {
    this.type = type;
    this.identifier = identifier;
    this.groupOrder = groupOrder;
  }
}
